Software/scan_key/AuthenticationSQL.py

```diff
@@ -45,30 +45,39 @@ def auth(id, pin, cursor):
 
     #SELECT * FROM USER WHERE t1String = id; (base way of getting info)
     cursor.execute("SELECT CUID, pin FROM USER WHERE t1String = " + id) #getting user w/ the id
-    data = cursor.fetchall() #fetching data into array
+    data = cursor.fetchone() #fetching data into array
 
     if(len(data)==0): #if there is no user with that data
         print("********** USER DOES NOT EXIST *********")
+        os.system("omxplayer deny.wav &")
         return(None) #no good
 
-    pinTest = data[0][1] #should've reversed this and idTest for styling purposes but this is the pin for the person with the id string
-    cuid = data[0][0] #and this is the cuid of the person with the id string
+    pinTest = data[0][1] #this is the pin for the person with the id string
+    cuid = data[0][0] #this is the cuid of the person with the id string
```

Comment:
I'm not 100% sure but I think swapping to fetchone should return a 1d array, not a 2d array.